android/src/main/java/com/clevertap/react/CleverTapModuleImpl.java

```diff
@@ -195,14 +114,15 @@ public void setLibrary(String libName, int libVersion) {
         if (cleverTap != null) {
             cleverTap.setCustomSdkVersion(libName, libVersion);
         }
+
     }
 
     public void setLocale(String locale) {
         CleverTapAPI cleverTap = getCleverTapAPI();
         if (cleverTap != null) {
             cleverTap.setLocale(locale);
         } else {
-            Log.e(TAG, ErrorMessages.CLEVERTAP_NOT_INITIALIZED.getErrorMessage());
+            Log.e(TAG, ErrorMessages.CLEVERTAP_NOT_INITIALIZED);
```

Comment:
is this change intentional?

Reply:
Yes, I've changed the ```ErrorMessages``` enum to an object with ```String``` constants. Those were only used as Strings by invoking ```.getErrorMessage()``` which I think is redundant.